config/settings.yml

```diff
@@ -0,0 +1,142 @@
+ps_reg_endpoint: 'http://registrar.example.edu'
```

Comment:
Can we only port settings on an as-needed basis? I suspect that we won't need many of these and if we bring them over now, they will never get cleaned out.

Reply:
Yeah, that's cool. I was originally thinking this would provide some info for what to port over, but I see your point, we can do that from the other side and try to keep this clean. I'll include only the few basic ones I need for now.